changelog/Microsoft.DirectoryServices.json

```
@@ -9873,6 +9873,68 @@
       "CreatedDateTime": "2021-11-15T20:42:47.116895Z",
       "WorkloadArea": "Identity and access",
       "SubArea": "Identity and sign-in"
+    },
+    {
+      "changelog": [
+        {
+          "ChangeList": [
+            {
+              "Id": "34b0a990-a3c9-4b70-92ac-a8ac93bf4860",
+              "ApiChange": "EnumType",
+              "ChangedApiName": "x509CertificateAuthenticationMode",
+              "ChangeType": "Addition",
+              "Description": "Added the [x509CertificateAuthenticationMode](https://docs.microsoft.com/en-us/?view=graph-rest-beta) resource",
+              "Target": "x509CertificateAuthenticationMode"
+            },
+            {
+              "Id": "34b0a990-a3c9-4b70-92ac-a8ac93bf4860",
+              "ApiChange": "EnumType",
+              "ChangedApiName": "x509CertificateRuleType",
+              "ChangeType": "Addition",
+              "Description": "Added the [x509CertificateRuleType](https://docs.microsoft.com/en-us/?view=graph-rest-beta) resource",
```

I hope that the previous descriptions weren't autogenerated as-was by Graph Studio?
```suggestion
              "ApiChange": "Enum type",
              "ChangedApiName": "x509CertificateAuthenticationMode",
              "ChangeType": "Addition",
              "Description": "Added the [x509CertificateAuthenticationMode](https://docs.microsoft.com/en-us/?view=graph-rest-beta) enumeration type.",
              "Target": "x509CertificateAuthenticationMode"
            },
            {
              "Id": "34b0a990-a3c9-4b70-92ac-a8ac93bf4860",
              "ApiChange": "Enum type",
              "ChangedApiName": "x509CertificateRuleType",
              "ChangeType": "Addition",
              "Description": "Added the [x509CertificateRuleType](https://docs.microsoft.com/en-us/?view=graph-rest-beta) enumeration type.",
```

---

It was auto generated when I submit PR to CSDL, I will make this suggested changes